src/engine.h

```diff
@@ -1,27 +1,24 @@
 // Declare this class here
-#include<iostream>
+#include <cstring>
+#include <iostream>
+#include <random>
 class RandomEngine {
     private:
          uint8_t length;
          bool symb;
+         char seq[100];
+         uint8_t mod=62;
+         const char alpha_num[63]="ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
+         const char alpha_num_sym[73]="ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789!@#$%^&*-_";
+         std::mt19937_64 *gen = new std::mt19937_64;
```

Comment:
No need to initialize objects here. Keep them uninitialized, and create objects in constructors. Something like this

In engine.h
```
std::mt19937_64 *gen;
```

in a  constructor

```
gen = new std::mt19937_64(seed);
```

Reply:
done.